router/core/websocket.go

```diff
@@ -792,7 +793,10 @@ func (h *WebSocketConnectionHandler) parseAndPlan(payload []byte) (*ParsedOperat
 	opContext.name = operationKit.parsedOperation.Request.OperationName
 	opContext.opType = operationKit.parsedOperation.Type
 
-	if blocked := h.operationBlocker.OperationIsBlocked(operationKit.parsedOperation); blocked != nil {
+	exprCtx := expr.RequestRootContext{}
+	exprCtx.LoadRequest(registration.clientRequest)
```

Comment:
It looks to me like at this point, we've not loaded Auth from the request into the exprCtx.

Reply:
Good point, a test is missing.